src/library_pthread.js

```diff
@@ -240,9 +240,9 @@ var LibraryPThread = {
 #if OFFSCREENCANVAS_SUPPORT
       if (typeof GL != 'undefined') {
         Object.assign(GL.offscreenCanvases, data.offscreenCanvases);
-        if (!Module['canvas'] && data.moduleCanvasId && GL.offscreenCanvases[data.moduleCanvasId]) {
-          Module['canvas'] = GL.offscreenCanvases[data.moduleCanvasId].offscreenCanvas;
-          Module['canvas'].id = data.moduleCanvasId;
+        if (!mainCanvas && data.moduleCanvasId && GL.offscreenCanvases[data.moduleCanvasId]) {
+          mainCanvas = GL.offscreenCanvases[data.moduleCanvasId].offscreenCanvas;
```

Comment:
Here is one such place where if user had existing code that accessed `Module['canvas']`, that code would no longer work?